test/cypress/integration/widgets/deployments_spec.js

```diff
@@ -12,11 +12,17 @@ describe('Deployments widget', () => {
             cy.get('.input.loading').should('not.exist');
         });
 
-    const actOnDeployment = (name, action) => {
-        searchForDeployment(name);
-        cy.contains('div.row', name).find('.deploymentActionsMenu').click();
+    const selectActionFromMenu = (deploymentId, menuClassName, action) => {
+        searchForDeployment(deploymentId);
+        cy.contains('div.row', deploymentId).find(menuClassName).click();
         cy.get('.popupMenu > .menu').contains(action).click();
     };
+    const actOnDeployment = (name, action) => {
```

Comment:
Maybe it's a good time to rename it to something more concrete, e.g.

```suggestion
    const executeDeploymentAction = (name, action) => {
```

What do you think?

Reply:
Make sense. Changed.